src/js/themes/base.js

```diff
@@ -723,6 +723,10 @@ export const generate = (baseSpacing = 24, scale = 6) => {
         //   background: undefined,
         // },
         // pad: undefined,
+        units: {
+          size: 'xsmall',
```

Comment:
I think we should default to `color = "text-xweak"` to align more with `theme.formField.info` and not change the size.

Reply:
This is the difference with color="text-xweak" and the default header size:
![image](https://user-images.githubusercontent.com/12054362/101587590-c1ddf980-39a1-11eb-86f5-e154e402f90c.png)